configure

```diff
@@ -108,6 +108,9 @@ while [ $# -ne 0 ]; do
       ;;
     --generator=*)
       CMakeGenerator="$optarg"
+      if [ "$CMakeGenerator" = "Ninja" ]; then
```

Comment:
Maybe also support `"ninja"` as lower-case alternative?

Reply:
Not supported by CMake.